server/routes.js

```diff
@@ -62,6 +65,18 @@ exports.upstreamQuintypeRoutes = function upstreamQuintypeRoutes(
     proxyReq.setHeader("Host", getClient(req.hostname).getHostname());
   });
 
+  const _sMaxAge = get(config, ["publisher", "upstreamRoutesSmaxage"], sMaxAge);
+
+  parseInt(_sMaxAge) &&
+    apiProxy.on("proxyRes", function (proxyRes, req) {
+      const pathName = get(req, ["originalUrl"], "").split("?")[0];
+      const checkForExcludeRoutes = excludeRoutes.every((item) => item !== pathName);
```

Comment:
why is `every` used, is this the right usecase?

Reply:
we can use `excludeRoutes.includes(pathName);`